Resources.md

```diff
@@ -0,0 +1,26 @@
+
+# Git:
+- Getting started with Git: See [Git Book](https://git-scm.com/book/en/v2) (chapters 1, 2, 3, 6)
+- GitHub flow: See [Introduction](https://guides.github.com/introduction/flow/) and [Fork a Repo](https://help.github.com/articles/fork-a-repo/)
+- GitHub Markdown: See [Markdown](https://guides.github.com/features/mastering-markdown/)
+
+# Git Software:
+- GUI Based: [SourceTree](https://www.sourcetreeapp.com/)
+- IDE Based: [VS Code](https://code.visualstudio.com/) + `GitLens`, `Git History` and `GitHub` extensions
+- CLI Based: [Git for Windows](https://git-scm.com/download/win) + [Posh Git](https://github.com/dahlbyk/posh-git)
+
```

Comment:
Consider [nbdime](https://github.com/jupyter/nbdime) for diffs inside the notebook